api/DockerfileDev

```diff
@@ -15,7 +15,7 @@ WORKDIR /app
 # We need a stub entry point for that
 COPY Cargo.toml Cargo.lock ./
 RUN mkdir src; echo "fn main() {}" > src/main.rs
-RUN cargo build --target x86_64-unknown-linux-musl
```

Comment:
Why this change?

Reply:
this didn't seem to break caching locally. is https://github.com/swissgeol/ngm/pull/834/commits/fe73733a95a7edf989d884dff798eb9ef19af84d working for you?